src/ql/ir/new_to_old.cc

```diff
@@ -486,10 +486,24 @@ void NewToOldConverter::convert_block(
                     // Handle the normal operands for custom instructions.
                     Operands ops;
                     for (const auto &ob : custom->instruction_type->template_operands) {
-                        ops.append(*this, ob);
+                        try {
+                            ops.append(*this, ob);
+                        } catch (utils::Exception &e) {
+                            e.add_context("name="+custom->instruction_type->name+", qubits="+ops.qubits.to_string());
```

Comment:
Shouldn't this be in append instead of here? Then appends needs knowledge of the instruction_time.
There are many more ops.append's where this context is not added.